bot/exts/backend/error_handler.py

```diff
@@ -167,28 +167,33 @@ async def try_get_tag(self, ctx: Context) -> None:
         by `on_command_error`, but the `invoked_from_error_handler` attribute will be added to
         the context to prevent infinite recursion in the case of a CommandNotFound exception.
         """
-        tags_get_command = self.bot.get_command("tags get")
-        if not tags_get_command:
-            log.debug("Not attempting to parse message as a tag as could not find `tags get` command.")
+        tags_cog = self.bot.get_cog("Tags")
+        if not tags_cog:
+            log.debug("Not attempting to parse message as a tag as could not find `Tags` cog.")
             return
+        tags_get_command = tags_cog.get_command_ctx
 
-        ctx.invoked_from_error_handler = True
+        maybe_tag_name = ctx.invoked_with
```

Comment:
Nitpick:
```suggestion
        tag_name = ctx.invoked_with
```
It's better this way IMO, and then you can check whether it exists or not like you'e doing in the following line.

Reply:
The following line only checks that it's not `None` (which should always be the case, but there's no harm in a check and it makes it so the type is correct later). It still might not be a tag name throughout it's usage, since this is run for *all* failed commands.

Reply:
Ahein, I though the `maybe` prefix was then validated by the following line, The name is still bugging me a bit though :p

Reply:
@shtlrs is it ok, if I resolve this now?

Reply:
It's a trivial matter so I guess we could leave it like that.